react-native-app/app.json

```diff
@@ -1,4 +1,6 @@
 {
   "name": "FitsnitchApp",
-  "displayName": "FitsnitchApp"
+  "displayName": "FitsnitchApp",
+  "versionCode": "2",
```

Comment:
The versions here don't match those in android/app/build.gradle. And what is the dpurpose having a versionCode separate from the versionName?

Reply:
So versionName and versionCode aren't needed here so I'm getting rid of it. But they are needed in the build.gradle. The versionName is just that--a name for the version that you deploy. But it can be whatever we wanted, whereas the versionCode needs to be a unique number associated with that build.

Reply:
Has this been resolved?

Reply:
Yes